docs/section5/Rule5-24.md

```diff
@@ -0,0 +1,93 @@
+
+# Envelope - Rule 5-24  
+
+**Rule ID:** 5-24  
+**Rule Description:** Vertical fenestration U-factors for residential, non-residential and semi-heated spaces in the baseline model must match the appropriate requirements in Table G3.4-1 through G3.4-8 for the appropriate WWR in the baseline RMR.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** Tables G3.4-1 to G3.4-8  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**  
+
+  1. data_lookup()
+  2. get_opaque_surface_type()
+  3. get_surface_conditioning_category()
+  4. get_scc_window_wall_ratios()
+
```

Comment:
@xingcx could we please rename get_scc_window_wall_ratio' to get_building_scc_window_wall_ratio'. Just to clarify that this is getting the WWR for the entire building and not for each building segment.

Reply:
Updated.